.github/workflows/main.yaml

```diff
@@ -0,0 +1,104 @@
+name: coinkite-tap-protocol
+
+on:
+  push:
```

Comment:
this will run on each push (no matter the branch) -> probably not needed this often
and for pull request on master branch